Xamarin.Forms.Core/Xamarin.Forms.Core.csproj

```diff
@@ -22,17 +22,8 @@
 	</ItemGroup>
 	<Import Project="..\Xamarin.Flex\Xamarin.Flex.projitems" Label="Shared" Condition="Exists('..\Xamarin.Flex\Xamarin.Flex.projitems')" />
 	<UsingTask TaskName="XFCorePostProcessor.Tasks.FixXFCoreAssembly" AssemblyFile="..\XFCorePostProcessor.Tasks\bin\Debug\net461\XFCorePostProcessor.Tasks.dll" />
-	<Target
-		Condition="$(DesignTimeBuild) != true AND $(BuildingProject) == true"
-		AfterTargets="AfterCompile"
-		Name="XFCorePostProcessor"
-		Inputs="$(IntermediateOutputPath)$(TargetFileName)"
-		Outputs="$(IntermediateOutputPath)XFCorePostProcessor.stamp">
-	    <Touch
-		    Files="$(IntermediateOutputPath)XFCorePostProcessor.stamp"
-		    AlwaysCreate="True" />
-	    <FixXFCoreAssembly
-		    Assembly="$(IntermediateOutputPath)$(TargetFileName)"
-		    ReferencePath="@(ReferencePath)" />
+	<Target Condition="$(DesignTimeBuild) != true AND $(BuildingProject) == true" AfterTargets="AfterCompile" Name="XFCorePostProcessor" Inputs="$(IntermediateOutputPath)$(TargetFileName)" Outputs="$(IntermediateOutputPath)XFCorePostProcessor.stamp">
```

Comment:
Dev16 auto changing project files

Reply:
you know you can pick the cadges you want to include in your commits, while doing `git add` ? use `git add -i`, or `git add -p`

Reply:
Yea :-/ but these are changes dev16 is going to make every time we open the SLN so it's easier to just get the project to a state of conforming